src/sentry/models/outbox.py

```diff
@@ -94,22 +89,274 @@ class OutboxCategory(IntEnum):
     SENTRY_APP_INSTALLATION_UPDATE = 10
     TEAM_UPDATE = 11
     ORGANIZATION_INTEGRATION_UPDATE = 12
-    ORGANIZATION_MEMBER_CREATE = 13  # Unused
+    UNUSUED_THREE = 13
     SEND_SIGNAL = 14
     ORGANIZATION_MAPPING_CUSTOMER_ID_UPDATE = 15
     ORGAUTHTOKEN_UPDATE = 16
     PROVISION_ORGANIZATION = 17
     POST_ORGANIZATION_PROVISION = 18
-    SEND_MODEL_SIGNAL = 19
+    UNUSED_ONE = 19
     DISABLE_AUTH_PROVIDER = 20
     RESET_IDP_FLAGS = 21
     MARK_INVALID_SSO = 22
     SUBSCRIPTION_UPDATE = 23
 
+    AUTH_PROVIDER_UPDATE = 24
+    AUTH_IDENTITY_UPDATE = 25
+
     @classmethod
     def as_choices(cls):
         return [(i.value, i.value) for i in cls]
 
+    def connect_region_model_updates(self, model: Type[ProcessUpdatesWithRegionOutboxes]) -> None:
```

Comment:
OutboxCategories now have several helper methods that are used directly into the Process mixin, but can also be used directly when you have a custom ad hoc wiring.